src/pages/contributing/index.md

```diff
@@ -77,6 +77,8 @@ Click Import and then select the `eclipse-format.xml` file in the `ide-config` d
 Next navigate to Java -> Code Style -> Organize Imports.
 Click Import and select the `eclipse.importorder` file in the `ide-config` directory.
 
+![Eclipse Code Formatter Settings](eclipse-code-formatter-settings.gif "Eclipse Code Formatter Plugin")
```

Comment:
Where is this gif hosted?

Reply:
Okay, it seems to be here https://github.com/jkubeio/jkube-website/pull/33 . Why did you create a separate PR for this?

Reply:
Mistakenly, I made separate PR's , Sorry will not repeat it again.